frontend/src/components/LatestMemes.tsx

```diff
@@ -24,7 +24,8 @@ export const LatestMemes = () => {
 
   useEffect(() => {
     if (data) {
-      const filteredData = data.filter((item) => item?.type === "image");
+      // filter out memes without url
+      const filteredData = data.filter((item) => item?.url); 
```

Comment:
Er det ikke bedre å gjøre dette i backenden? Dersom det bare ønskes å vise posts med media?

Reply:
Jo, hadde vært best

Reply:
Bare legg til dette i /latest-memes SQL-querrien: " WHERE ChannelName = @ChannelName AND Url IS NOT NULL"

Reply:
![image](https://github.com/user-attachments/assets/6290225f-aa6e-46dd-87d7-2bff969372a5)